README.md

```diff
@@ -30,10 +31,51 @@ jobs:
       - name: Checkout Repo
         uses: actions/checkout@v3
       - name: Secret Scanner
-        uses: secret-scanner/action@0.0.2
+        uses: secret-scanner/action@main #or latest release
 ```
 
-You can also pass arguments to `detect-secrets` by using `detect-secret-additional-args`. For information on the arguments that you can pass, visit [Yelp/detect-secrets#filters](https://github.com/Yelp/detect-secrets#filters). For example:
+### Ignoring false positives
+Often you might have strings that the secret scanner determines are secrets, but are actually harmless. Examples of these might be:
+1. Docker tags
+2. Git commit SHA's
+3. Randomly generated base64 strings
+
+For these cases, it is useful to ignore certain files, lines, or "secrets". You can do this using the files:
+- `.github/actions/secret-scanner/excluded_files_regex.txt`
```

Comment:
I'd suggest this as the filename ("txt" doesn't tell anyone anything):
```suggestion
# File: .github/actions/secret-scanner/excluded_files.patterns
```

(And similar for the others)